servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1071,6 +1104,17 @@ public String getUfsAddress() {
     return mWhitelist.getList();
   }
 
+  public List<Long> getLostFiles() {
+    Set<Long> lostFiles = Sets.newHashSet();
```

Comment:
why not make `lostFiles` an `ArrayList`, and return it directly? Currently, you are doing one more conversion from `HashSet` to `ArrayList`